kod/object/active/logghost.kod

```diff
@@ -620,6 +632,9 @@ messages:
       piShield_translation = Send(poGhostedPlayer,@GetShieldTranslation);
       oWeapon = Send(poGhostedPlayer,@LookUpPlayerWeapon);
 
+      plOverlays = Send(poGhostedPlayer,@GetOverlays);
```

Comment:
I'm on the fence about whether GetOverlays() is a good idea.  It exposes the internal overlays to anyone, breaking encapsulation and potentially leading to some code chaos if someone later comes along and starts modifying the list outside of Player.  I wonder if there are other options.